ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ArtifactRegistryIntegrationForm.tsx

```diff
@@ -201,27 +219,48 @@ function ArtifactRegistryIntegrationForm({
                             />
                         </FormLabelGroup>
                     )}
-                    <FormLabelGroup
-                        label="Service account key (JSON)"
-                        isRequired={values.updatePassword}
-                        fieldId="config.google.serviceAccount"
-                        touched={touched}
-                        errors={errors}
-                    >
-                        <TextArea
+                    {isCloudCredentialsEnabled && (
+                        <FormLabelGroup
+                            fieldId="config.google.wifEnabled"
+                            touched={touched}
+                            errors={errors}
+                        >
+                            <Checkbox
+                                label="Use workload identity"
+                                id="config.google.wifEnabled"
+                                aria-label="enable wif"
```

Comment:
Someone else might be able to suggest pro and con for `aria-label` screen reader attribute to have same value as `label` attribute?

Reply:
I don't think the `aria-label` is needed here, since there is a `label` prop.

I think we have to use the aria-label on things like the "select" row or all checkboxes in a table, and maybe that pattern got copy-pasted to a form checkbox at some point.

You should remove the aria-label and test the rendered markup with Axe to double-check that we don't need it.

Reply:
This is the `axe` output without the `aria-label`
```
  Violation of "document-title" with 1 occurrences!
    Ensures each HTML document contains a non-empty <title> element. Correct invalid elements at:
     - html
    For details, see: https://dequeuniversity.com/rules/axe/3.5/document-title

  Violation of "html-has-lang" with 1 occurrences!
    Ensures every HTML document has a lang attribute. Correct invalid elements at:
     - html
    For details, see: https://dequeuniversity.com/rules/axe/3.5/html-has-lang

  Violation of "landmark-one-main" with 1 occurrences!
    Ensures the document has a main landmark. Correct invalid elements at:
     - html
    For details, see: https://dequeuniversity.com/rules/axe/3.5/landmark-one-main

  Violation of "page-has-heading-one" with 1 occurrences!
    Ensure that the page, or at least one of its frames contains a level-one heading. Correct invalid elements at:
     - html
    For details, see: https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one

4 Accessibility issues detected.
```

Reply:
Some odd errors (we have a `lang` and a `<title>`, but I don't see an error for missing aria-label, so the <label> element is good enough.